pkg/ibmcsidriver/server.go

```diff
@@ -137,6 +158,35 @@ func (s *nonBlockingGRPCServer) Setup(endpoint string, ids csi.IdentityServer, c
 	return listener, nil
 }
 
+func (f *realFileOps) Chown(name string, uid, gid int) error {
```

Comment:
can we have a separate file for file ops, I would suggest to use socketPermission as interface name also see if we can make these local method

Reply:
done